### PR TITLE
Simplify reporting error for unsupported `createCredentials` api

### PR DESCRIPTION
**Pull request by ForbesLindesay**
Using `each` for just one string seems like a lot of un-necessary overhead.  It also seemed a little pointless to use a separate `error` function when we only call it from that one location.

### CODE REVIEW

**Comment by dominictarr** [comment]
:+1: yes, because this is now the only thing left!

**Comment by ForbesLindesay** [comment]
Indeed, it's a fantastic achievement :)

File: index.js [hunk 1]
```diff
@@ -50,10 +37,10 @@ require('create-ecdh/inject')(module.exports, exports);
...
-], function (name) {
-  exports[name] = function () {
-    error('sorry,', name, 'is not implemented yet')
-  }
-})
+exports.createCredentials = function () {
+  throw new Error([
+    'sorry, createCredentials is not implemented yet',
+    'we accept pull requests',
+    'http://github.com/dominictarr/crypto-browserify'
```
Comment (calvinmetcalf) [inline index.js]: this isn't the repo any more

File: index.js [hunk 2]
```diff
@@ -50,10 +37,10 @@ require('create-ecdh/inject')(module.exports, exports);
...
 
 // the least I can do is make error messages for the rest of the node.js/crypto api.
-each([
-  'createCredentials'
-], function (name) {
-  exports[name] = function () {
-    error('sorry,', name, 'is not implemented yet')
-  }
-})
+exports.createCredentials = function () {
```
Comment (dcousens) [inline index.js]: `function createCredentials () {` if possible,  helps with debugging :)

**Comment by dcousens** [comment]
ACK after URL fix up and function naming.

**Comment by calvinmetcalf** [comment]
bad news guys, they just added privateEncrypt and publicDecrypt to the api lets see how quick I can add them to publicEncrypt

**Comment by dcousens** [comment]
Merged in #121

**Pull request closed without merging**